src/viam/sdk/common/private/proto_utils.hpp

```diff
@@ -17,7 +17,7 @@ void vecToRepeatedPtr(const std::vector<Src>& vec, google::protobuf::RepeatedPtr
     dest.Clear();
     dest.Reserve(vec.size());
     for (auto& x : vec) {
-        *dest.Add() = x.to_proto();
+        *dest.Add() = v2::to_proto(x);
```

Comment:
Would `v2` be dropped out before merge?

Reply:
`v2` gets dropped once I've done the other PRs that include other instances of `to_proto` and `from_proto`!